components/PromptRemove.vue

```diff
@@ -5,14 +5,21 @@ import Card from '@/components/Card';
 import Checkbox from '@/components/form/Checkbox';
 import { alternateLabel } from '@/utils/platform';
 import { uniq } from '@/utils/array';
+import { sortBy } from '@/utils/sort';
 import AsyncButton from '@/components/AsyncButton';
-import { CATALOG as CATALOG_ANNOTATIONS } from '@/config/labels-annotations';
-import { CATALOG } from '@/config/types';
+import { CATALOG as CATALOG_ANNOTATIONS, PROJECT } from '@/config/labels-annotations';
+import { CATALOG, MANAGEMENT, NAMESPACE } from '@/config/types';
 
 export default {
   components: {
     Card, Checkbox, AsyncButton
   },
+
+  async fetch() {
```

Comment:
I know this is only draft, but we should not add functionality into the generic promptRemove dialog that is specific to a given resource.

There is a folder `promptRemove` where you can add a component for a given resource and this will extend the generic prompt remove dialog without having code in there that affects all resources.